src/features/groups/components/GroupSingleHeaderDropdownMenuTriggerButton/index.tsx

```diff
@@ -22,18 +26,51 @@ export const GroupSingleHeaderDropdownMenuTriggerButton = ({
   groupId,
   currentGroupName,
 }: IGroupSingleHeaderDropdownMenuTriggerButton) => {
-  const [isDropdownMenuOpen, setIsDropdownMenuOpen] = useState(false);
+  const [isRenameGroupDrawerOpen, setIsRenameGroupDrawerOpen] = useState(false);
+  const [isDeleteGroupDialogOpen, setIsDeleteGroupDialogOpen] = useState(false);
```

Comment:
When we need to have more dropdown menu options, consider replacing this part using `useReducer`.